airbyte-integrations/connectors/source-posthog/integration_tests/expected_records.json

```diff
@@ -0,0 +1,659 @@
+{"stream":"projects","data":{"id":2331,"uuid":"0178cc4f-df47-0000-c607-ca58c009c7df","organization":"0178cc4f-de6a-0000-091b-f4791e6e02f7","api_token":"r4uy4lgjKEGSkSbUdyr3AydH1R7EYfEtg25A7B58-N0","name":"Default Project","completed_snippet_onboarding":true,"ingested_event":true,"is_demo":false,"timezone":"UTC","access_control":false,"effective_membership_level":15},"emitted_at":1668174290927}
```

Comment:
were those expected records produced from the python connector? It would be nice to verify we don't introduce bugs as part of this migration